docs/en/integrations/data-visualization/tableau/tableau-and-clickhouse.md

```diff
@@ -0,0 +1,172 @@
+---
+sidebar_label: Tableau Desktop
+sidebar_position: 1
+slug: /en/integrations/tableau
+keywords: [ clickhouse, tableau, connect, integrate, ui ]
+description: Tableau can use ClickHouse databases and tables as a data source.
+---
+import TOCInline from '@theme/TOCInline';
+import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_http.mdx';
+
+# Connecting Tableau to ClickHouse
+
+ClickHouse offers an official Tableau Connector, featured on
+the [Tableau Exchange](https://exchange.tableau.com/products/1064).
+The connector is based on ClickHouse's advanced [JDBC driver](/docs/en/integrations/java/jdbc-driver).
+
+With this connector, Tableau integrates ClickHouse databases and tables as data sources. To enable this functionality,
+follow the setup guide bellow.
+
+
+<TOCInline toc={toc}/>
+
+## Setup Required Prior Usage
+
+
+1. Gather your connection details
+   <ConnectionDetails />
+
+2. Download and install  <a href="https://www.tableau.com/products/desktop/download" target="_blank">Tableau
+   desktop</a>.
+3. Follow `clickhouse-tableau-connector-jdbc` instructions to download the compatible version
+   of <a href="https://github.com/ClickHouse/clickhouse-java/releases/" target="_blank">ClickHouse JDBC driver</a>.
+
+:::note
+Make sure you download the **clickhouse-jdbc-x.x.x-shaded.jar** JAR file. Currently, we recommended using version `0.4.6`.
+:::
+
+4. Store the JDBC driver in the following folder (based on your OS, if the folder doesn't exist you can create it):
+    - macOS: `~/Library/Tableau/Drivers`
+    - Windows: `C:\Program Files\Tableau\Drivers`
+5. Configure a ClickHouse data source in Tableau and start building data visualizations!
+
+## Configure a ClickHouse data source in Tableau
+
+Now that you have the `clickhouse-jdbc` driver installed and set, let's see how to define a data
+source in Tableau that connects to the **TPCD** database in ClickHouse.
+
+1. Start Tableau. (If you already had it running, then restart it.)
+
+2. From the left-side menu, click on **More** under the **To a Server** section. Search for **ClickHouse by ClickHouse** in the available connectors list:
+
+   ![ClickHouse JDBC](../images/tableau_connecttoserver.png)
+<br/>
+
+:::note
+Don't see the **ClickHouse by ClickHouse** connector in your connectors list? It might be related to an old Tableau Desktop version.
+To solve that, consider upgrading your Tableau Desktop application, or [install the connector manually](#install-the-connector-manually).
```

Comment:
I removed my Tableau Desktop and installed version `2023.3.11` and it seems our connector is there. I believe they fetch all available connectors from some internal API (because the connector itself isn't shipped with the distribution, but only has a link to download from the Tableau Exchange site) - but again, I'm not 100% sure.

Reply:
ok, let's them keep it as is